src/backend/core/tests/monitoring/test_probes_cidr.py

```diff
@@ -0,0 +1,179 @@
+"""Test liveness and readiness CIDR protected probes of impress's core app."""
+
+import os
+from unittest.mock import patch
+
+from django.test import TestCase
+
+
+class ProbesCidrProtectionTest(TestCase):
```

Comment:
We are not using django TestCase in this project. We are using pytest django instead. See how other tests are implemented